File: PythonJob/TimeLine.py
from Zaman import Zaman
class TimeLine(object):
    """description of class"""


    def __init__(self, s,f):
        self.DayStart = s
        self.DayFinish = f
        return super(TimeLine, self).__init__()
    
    def MaksJobLength(self,value=0):
        self._MaksJobLength = value
                
    def MinJobLength(self,value=0):
        self._MinJobLength = value

    def MaksJL(self):
        return self._MaksJobLength
    
    def MinJL(self):
        return self._MinJobLength
    
    def DayLengthMinute(self):
        return Zaman.ZamanFarki(self.DayFinish,self.DayStart);

File: Job/TimeLine.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Job
{
    public class TimeLine
    {

        public TimeSpan DayStart { get; set; }
        public TimeSpan DayFinish { get; set; }
        public int MaksJobLength { get; set; } = 30;
        public int MinJobLength { get; set; } = 10;
        public int DayLengthMinute
        {
            get
            {
                return (int)((DayFinish - DayStart).TotalMinutes);
            }
        }





        public TimeLine(TimeSpan s, TimeSpan f)
        {
            DayStart = s;
            DayFinish = f;
        }



    }
}
